scripting/get5_apistats.sp

```diff
@@ -138,11 +141,18 @@ static Handle CreateRequest(EHTTPMethod httpMethod, const char[] apiMethod, any:
 
 int RequestCallback(Handle request, bool failure, bool requestSuccessful,
                     EHTTPStatusCode statusCode) {
-  if (failure || !requestSuccessful) {
-    LogError("API request failed, HTTP status code = %d", statusCode);
-    char response[1024];
-    SteamWorks_GetHTTPResponseBodyData(request, response, sizeof(response));
-    LogError(response);
+  int status = view_as<int>(statusCode);
+  if (failure || !requestSuccessful || status >= 300 || status < 200) {
```

Comment:
If `failure` or `!requestSuccesful`, you should not check for HTTP status code or attempt to read the body. This needs to be broken into two parts.